### DISCUSSION TITLE
stepper_z is 200 ? [Kobra2Neo]

### DISCUSSION THREAD

**Discussion by bimawa** [1coderookie/Klipper4Kobra2series#7] [General]
I found one moment. Is stepper_z
```
[stepper_z]
step_pin: PB0
dir_pin: !PB1
enable_pin: !PA15
microsteps: 16
rotation_distance: 8
endstop_pin: probe:z_virtual_endstop
position_min: -2
position_max: 200
homing_speed: 15
second_homing_speed: 2
```
But kobra 2 neo has height 250mm
Is it a mistake or is this choice based on some kind of nuance?

**Comment by 1coderookie** [comment]
I didn't make that cfg, so I can't say *why* it has been set like that, but imho you should be able to set that to 250 max (or maybe 245), yes.  
Actually, you also should be able to set X and Y to 230 max, since that's the actual size of the bed as you can see [here](https://1coderookie.github.io/Kobra2NeoInsights/hardware/bed/).   

If you change any of these max values for the axes, always jog the head in it's max position carefully first then, don't just enter the command to drive it to the max position then.  
Since we don't have any maximum limit switches, you don't want to risk driving any axis beyond it's physical limit.  
So even tho the abovementioned settings should work just fine, be careful when driving the axes to their limits the very first time to check if those settings really do work safely.

**Reply by bimawa** [reply]
wow, Ok thanks!
